src/components/Filter/Filter.test.tsx

```diff
@@ -0,0 +1,13 @@
+import { render } from '@testing-library/react';
+import React from 'react';
+import Filter from './Filter';
+
+describe('Filter component', () => {
+  it('Should be show filter component', () => {
```

Comment:
I like to write/read my tests like this:

`describe`: What the whole thing I'm testing is. `Filter Component`, `Toggler`, `Tab`, etc. 
`it`: Completes what the "thing" is expected to do, and be read in a sentence like `{ describe }` `{it}`: 
  - `Filter Component` + `should be shown`, 
  - `Filter Component` + `should display one <li> per string`

Reply:
Fixed